core/src/main/java/com/linecorp/armeria/client/Client.java

```diff
@@ -71,6 +73,7 @@
      * @see ClientFactory#unwrap(Object, Class)
      * @see Unwrappable
      */
+    @Nullable
```

Comment:
`@Nullable` is an inherited annotation, which means you do not have to specify it when the method you are overriding already has it, i.e. this is false positive.

Reply:
Agreed. I reverted all `@Nullable` in override methods.